packages/zoe/src/zoe.js

```diff
@@ -673,17 +928,24 @@ const makeZoe = (additionalEndowments = {}) => {
               proposal,
               currentAllocation: arrayToObj(amountsArray, userKeywords),
             };
+            // TODO split inviteHandle and offerHandle.
```

Comment:
We should tackle this before merging, I think. Is that ok?

Reply:
I think so. Outside of zoe.js and state.js, I think I changed inviteHandle and offerHandle names to where they should be, anticipating this change. If you see anywhere that I use the wrong handle name type please let me know. It'll make this next step easier.

Reply:
Ref #889

Reply:
Before we can do this (closing #889 as well as deleting `makeGetInviteHandle` etc), we need to resolve the issue revealed by test-simpleExchange explained above --- where the offerHandle often only becomes available after the offer is no longer active. Is this a problem?

Reply:
Due to your suggestion, test-simpleExchange is fixes, so nothing blocks us from fixing #889 in this PR or the next one. If we need it fixed in this one, please let me know. Otherwise I'll leave it to the next one.